src/sysfiles.c

```diff
@@ -585,7 +585,7 @@ void SyBufSetEOF(Int fid)
 
 /****************************************************************************
 **
-*F  SyFopen( <name>, <mode> ) . . . . . . . .  open the file with name <name>
+*F  SyFopen( <name>, <mode>, <transcompress> ) open the file with name <name>
```

Comment:
Please make the same change in the header file (and add the sentence about `transcompress`).